build.gradle

```diff
@@ -78,6 +79,8 @@ dependencies {
     testCompile group: 'org.neo4j', name: 'neo4j-causal-clustering', version:neo4jVersionEffective, classifier: "tests"
     testCompile group: 'org.neo4j', name: 'neo4j-kernel', version:neo4jVersionEffective, classifier: "tests"
     testCompile group: 'org.neo4j', name: 'neo4j-io', version:neo4jVersionEffective, classifier: "tests"
+    testCompile group: 'com.oracle', name: 'ojdbc7', version: '12.1.0.2'
+    testCompile group: 'postgresql', name: 'postgresql', version: '9.4.1208-jdbc42-atlassian-hosted'
```

Comment:
what is this? '9.4.1208-jdbc42-atlassian-hosted'
there is a maven central driver for postgresql?

Reply:
yes there is, sorry,  we change it